templates/core/features/index.html

```diff
@@ -4,314 +4,276 @@
-      {{ image (
-        url="https://assets.ubuntu.com/v1/4563fd30-UC20_advanced_security_features_2.svg",
-        alt="",
-        width="177",
-        height="150",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
+    <div class="col">
+      <div class="p-equal-height-row">
+        <div class="p-equal-height-row__col">
+          <h3 class="p-heading--5 p-equal-height-row__item">
+            <a href="https://snapcraft.io/docs/snap-confinement">Strict confinement&nbsp;&rsaquo;</a>
+          </h3>
+          <p class="p-equal-height-row__item">A whole new armoury of Linux security capabilities to ensure strict confinement. Leverage a clean separation between the kernel, OS image and the developers’ applications, providing a secure and immutable OS.</p>
+        </div>
+        <div class="p-equal-height-row__col">
+          <h3 class="p-heading--5 p-equal-height-row__item">
+            <a  href="/core/features/full-disk-encryption">Full disk encryption&nbsp;&rsaquo;</a>
+          </h3>
+          <p class="p-equal-height-row__item">Disks are locked with private key-based cryptography. Private keys for hardware, TPM and other layers are securely stored. Symmetric key encryption is enabled by the use of specialised software-enabled stores.</p>
+        </div>
+        <div class="p-equal-height-row__col">
+          <h3 class="p-heading--5 p-equal-height-row__item">
+            <a href="/core/features/secure-boot">Secure boot&nbsp;&rsaquo;</a>
+          </h3>
+          <p class="p-equal-height-row__item">Each component in the boot sequence cryptographically validates the authenticity of the subsequent component. Every component is measured before it’s loaded in the runtime memory space.</p>
+        </div>
+      </div>
     </div>
   </div>
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>OTA updates</h2>
-      <p>Over the air updates for Linux done right </p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Transactional updates for reliability</li>
-        <li class="p-list__item is-ticked">Diffs only to minimise network traffic</li>
-        <li class="p-list__item is-ticked">Digital signatures to guarantee integrity and provenance</li>
-      </ul>
-      <p>
-        <a href="/core/features/ota-updates">Learn more about OTA updates&nbsp;&rsaquo;</a>
-      </p>
-    </div>
-    <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
+  <div class="row">
+    <div class="col-9 col-start-large-4 col-medium-6">
+      <div>
         {{ image (
-          url="https://assets.ubuntu.com/v1/a84416b2-IoT-gateways-on-ubuntu.svg",
-          alt="",
-          width="263",
-          height="150",
+          url="https://assets.ubuntu.com/v1/80d597dd-security.jpg",
+          alt="Three big robotic arms in a factory",
+          width="916",
```

Comment:
Could you adjust the size so that it lines up with the grid like on the design? 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/a9edfb40-8787-46ab-9c6a-691eca5c0c44)
vs 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/d0595e6c-f9ef-47bc-af96-20599abe7919)

Reply:
Already mentioned that the image has a size of 916px on the design which Lyubo told me to use

Reply:
Did he confirm though? It still doesn't match the designs on Figma.

Reply:
We shouldn't be taking things like image sizes from figma so literally IMO, they very often don't translate correctly on the actual page. There've been a number of times where an image will be one size on figma but to get it to actually look the same way in the code the values might have to be tweaked a bit. The true north for all of these pages has always been to match the design, not necessarily to match _how_ the design was built on figma if that makes sense